### PR TITLE
add ARCHI NEWSAN board

### PR DESCRIPTION
**Pull request by EricAlmendras**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -386,3 +386,21 @@
...
+	url = https://github.com/adafruit/Adafruit_CircuitPython_Pixel_Framebuf
+[submodule "frozen/Adafruit_CircuitPython_Led_Animation"]
+	path = frozen/Adafruit_CircuitPython_Led_Animation
+	url = https://github.com/adafruit/Adafruit_CircuitPython_Led_Animation
+[submodule "frozen/Adafruit_CircuitPython_Neopixel"]
+	path = frozen/Adafruit_CircuitPython_Neopixel
+	url = https://github.com/adafruit/Adafruit_CircuitPython_Neopixel
+[submodule "frozen/Adafruit_CircuitPython_MatrixKeypad"]
+	path = frozen/Adafruit_CircuitPython_MatrixKeypad
+	url = https://github.com/adafruit/Adafruit_CircuitPython_MatrixKeypad
```
Comment (dhalbert) [inline .gitmodules:406]: You are duplicating some modules that are already frozen. Please check in `.gitmodules`.

Comment (dhalbert) [inline .gitmodules:406]: For `MatrixKeypad`, consider using the `keypad` module instead.

File: ports/raspberrypi/boards/archi/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,24 @@
+USB_VID = 0x1043
+USB_PID = 0x2E8A
```
Comment (dhalbert) [inline ports/raspberrypi/boards/archi/mpconfigboard.mk:2]: Where did you get this VID and PID? Are you the manufacturer of this board?

**Review by dhalbert — CHANGES_REQUESTED** [review]
The content of the headers of the `board` files has changed. Please see the tip of `main` for the new standard headers.

Please rebase your branch from `main`. You're changing the commits for a number of existing frozen modules.

File: ports/raspberrypi/boards/archi/mpconfigboard.mk (hunk 2 above)
Comment (EricAlmendras) [reply ports/raspberrypi/boards/archi/mpconfigboard.mk:2]: Yes, I am part of the team that developed that board, that VID and PID are the ones that gave us the raspberrypi foundation for this board

Comment (dhalbert) [reply ports/raspberrypi/boards/archi/mpconfigboard.mk:2]: Aha, I see. But I think you've reversed the VID and PID. The RPi **VID** is 0x2E8A. (https://github.com/raspberrypi/usb-pid)

File: .gitmodules (hunk 1 above)
Comment (dhalbert) [reply .gitmodules:406]: https://learn.adafruit.com/key-pad-matrix-scanning-in-circuitpython

File: ports/raspberrypi/boards/archi/mpconfigboard.mk (hunk 2 above)
Comment (EricAlmendras) [reply ports/raspberrypi/boards/archi/mpconfigboard.mk:2]: Thanks, I hadn't noticed that. :)

**Pull request closed without merging**